### PR TITLE
Misc. non-critical cleanup and refactor

### PR DESCRIPTION
**Pull request by maflcko**

### CODE REVIEW

File: contrib/debian/examples/bitcoin.conf [hunk 1]
```diff
@@ -67,9 +67,8 @@
 # This option can be specified multiple times (default: bind to all interfaces)
 #rpcbind=<addr>
 
-# You must set rpcuser and rpcpassword to secure the JSON-RPC api
-#rpcuser=Ulysseys
-#rpcpassword=YourSuperGreatPasswordNumber_DO_NOT_USE_THIS_OR_YOU_WILL_GET_ROBBED_385593
+# You must set rpcauth to secure the JSON-RPC api
+#rpcauth=<result from ./share/rpcuser/rpcuser.py User01>
```
Comment (MarcoFalke) [inline contrib/debian/examples/bitcoin.conf]: Anyone knows where I could find this folder? Doesn't look like it's in the ppa...

File: src/qt/intro.cpp [hunk 2]
```diff
@@ -17,7 +17,7 @@
 
 /* Minimum free space (in bytes) needed for data directory */
 static const uint64_t GB_BYTES = 1000000000LL;
-static const uint64_t BLOCK_CHAIN_SIZE = 20LL * GB_BYTES;
+static const uint64_t BLOCK_CHAIN_SIZE = 80LL * GB_BYTES;
```
Comment (jonasschnelli) [inline src/qt/intro.cpp]: What if you start (first time) bitcoin-qt with `-prune=550`?

File: src/qt/forms/overviewpage.ui [hunk 3]
```diff
@@ -20,7 +20,7 @@
       <bool>false</bool>
      </property>
      <property name="styleSheet">
-      <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0, stop:0 #F0D0A0, stop:1 #F8D488); color:#000000;</string>
+      <string notr="true">QLabel { background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0, stop:0 #F0D0A0, stop:1 #F8D488); color:#000000; }</string>
```
Comment (jonasschnelli) [inline src/qt/forms/overviewpage.ui]: What's the purpose of limiting the styles to `QLabel`?

Comment (jonasschnelli) [inline src/qt/forms/overviewpage.ui]: Oh.. wait... this looks after one of my stashed commits.. :) Never mind.

File: src/qt/intro.cpp (hunk 2 above)
Comment (MarcoFalke) [inline src/qt/intro.cpp]: @jonasschnelli Created another pull to cover this

**Pull request closed without merging**